### PR TITLE
updates the step-1 and step-2 with appropriate information in the Confirmation Pages

### PR DESCRIPTION
**Pull request by sangeet259**
1. Adds how to add the necessary verifier text in the ERP question!
2. The mail by Leprechaun lands in spam folders sometimes, this PR adds this info in the step-2 page.

### CODE REVIEW

File: templates/step1_complete.tmpl.html [hunk 1]
```diff
@@ -22,6 +22,9 @@ <h3>
                 <h3>
                     Visit the link inside that email to complete your
                     authentication!
+                    </br>
+                    Note : Check your spam and promotions if the mail is not in your primary space!
```
Comment (icyflame) [inline templates/step1_complete.tmpl.html]: Put `Note:` under `<b></b>`

File: templates/begin_auth.tmpl.html [hunk 2]
```diff
@@ -17,8 +17,16 @@ <h1 class="display-3">
...
-                    Add this to the text of one of your security questions:
-                    {{.Verifier}}
+                    Add this to the text of one of your security questions: {{.Verifier}}
+                    <br>
+                    To do that :
+                    <ol>
+                        <li>Open ERP</li>
+                        <li>Click on forgot password</li>
+                        <li>Click on Change Security questions</li>
+                        <li>Enter 3 questions, anyone of which must be {{.Verifier}}. It's answer can be anything.</li>
```
Comment (icyflame) [inline templates/begin_auth.tmpl.html]:
```html
<li>
    any one of the three questions must contain {{.Verifier}}.
    <br>
    <b>Pro-Tip:</b> Enter your old question followed by {{.Verifier}}!
    <br>
    Eg: Original question: <i>What was the name of your first pet?</i>
    <br>
    Required question: <i>What was the name of your first pet? {{.Verifier}}</i>
</li>
```

File: templates/begin_auth.tmpl.html [hunk 3]
```diff
@@ -17,8 +17,16 @@ <h1 class="display-3">
                     Authentication Flow has begun!
                 </h1>
                 <h3>
-                    Add this to the text of one of your security questions:
-                    {{.Verifier}}
+                    Add this to the text of one of your security questions: {{.Verifier}}
+                    <br>
+                    To do that :
```
Comment (icyflame) [inline templates/begin_auth.tmpl.html]: We shouldn't put this information inside `div.jumbotron`. Put it after this div.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by sangeet259** [comment]
@icyflame changes made

**Comment by icyflame** [comment]
Hey @sangeet259, thanks for your changes. I have merged them in. I put the guide text outside the jumbotron and added a note about the guide in the jumbotron.

There are definitely other enhancements that can be made on the user-facing side of this module. If you have the time, you can review the other templates in this flow and send patches as you see fit. I will be happy to accept them. :slightly_smiling_face:

**Comment by sangeet259** [comment]
Sure @icyflame :+1:

**Pull request closed without merging**